recipes/logr/all/conanfile.py

```diff
@@ -0,0 +1,97 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+class LogrConan(ConanFile):
+    name = "logr"
+    license = "BSD 3-Clause License"
```

Comment:
```suggestion
    license = "BSD-3-Clause"
```

Use the short official form: https://spdx.org/licenses/BSD-3-Clause.html